### ISSUE TITLE
BUG-MAJOR_AMICS_PROCESS_BATCH_Touchup steps not applied during batch

### ISSUE THREAD

**Issue by Matt-Power** [Bruker-Amics/bugs_features#357]
v3.3.0.1660

This will be another really useful addition, thank you but...:

Unfortunately, the steps do not seem to be applied during batch processes. This is the case for both mapped measurements and for mixes of mapped and segmented measurements. Not tried on segmented measurements only though. The steps also do not appear in the single sample reclassify dialogue box (either for a refresh or for a reclassify against ore type). This means that the steps have to be recreated every time and applied individually to each measurement which, unfortunately, defeats the object.

**Comment by jonathan-knapp** [comment]
@Matt-Power this item has been added to the "In Progress" queue.

**Comment by ChiLyMan** [comment]
Touch Up is still not being applied during reclassification or batch reclassification.
This is the original image without any touch-up
![image.png](https://images.zenhubusercontent.com/62f264fb4831bda01fb7d454/cd9f7be9-2f10-4457-b775-960935c7ec1a)
I then apply a broad touch up to convert the all Lepidolite to Apatite, changing the image to look like this:-
![image.png](https://images.zenhubusercontent.com/62f264fb4831bda01fb7d454/9365f714-cd66-4565-8d84-1f257c2ae822)
I then reclassify without any touch up to revert back to the original data containing Lepidolite, then reclassify again with the touch-up step enabled.
![image.png](https://images.zenhubusercontent.com/62f264fb4831bda01fb7d454/b404071c-76c5-4846-9d92-a57997be9df8)
But the reclassified and touch up data does not seem to have the touch-up applied. The image below shows the result after this reclassficaition + touch up, with the blue lepidolite still present and not converted to Apatite:-
![image.png](https://images.zenhubusercontent.com/62f264fb4831bda01fb7d454/0dbade2a-2cdd-49c3-9861-454b5d1a62ba)

Got the same result when I did this via a batch process as well.

**Comment by ChiLyMan** [comment]
Testing version 1934:
Touch up steps are still not being applied during batch processing, even with the step selected as part of the material classification dictionary. This is the case for both Matrix Mapping and Segmentation Mapping measurements, or any combination of these.
![image.png](https://images.zenhubusercontent.com/62f264fb4831bda01fb7d454/a89d09d8-d406-4415-aa87-7921b554e092)
Maybe the Batch processing function still has not included the steps into it like we have for the normal processing window?

Also, touch up steps are not being applied to Matrix Mapping samples even in single sample reclassification mode, even with the selected touch up checked boxed enabled. Tested this on Segmentation Mapping data and touch up steps are being applied to this type of analysis.
![image.png](https://images.zenhubusercontent.com/62f264fb4831bda01fb7d454/e01db20a-39ec-4527-966d-d6aafd09325d)
I also tried converting the Matrix Mapping measurement to Segmentation Mapping using the Particulate function and saved the data to the file (not just in memory) and restarted AMICS and Touch Up steps are still not being applied during reclassification (batch or single).
NOTE: All samples were run as Granulated type samples.
